docs/reference/protocol-spec.md

```diff
@@ -365,11 +364,20 @@ Here's the encoding of the number 1.23:
 
     ,1.23\r\n
 
-The floating point number must start with a digit, even if it is zero (0).
-Exponential notation is not supported.
+A floating point number may be made up of up to three parts:
+
+    <integral-part>[.<fractional-part>[<exponent-part]]
+
+The floating point number always begins with an integral.
+The integral part is made of one or more decimal digits.
+
+A fractional part may follow the integral.
+The fractional part begins with a dot (`.`), followed by one or more decimal digits.
 
-However, the decimal part is optional.
-The integer value of ten (10) can, therefore, be RESP-encoded both as an integer as well as a double:
+An exponent part may follow the fractional.
```

Comment:
Perhaps rephrase slightly? The exponent part can also follow the integral part, if there are no decimal fractions.

Reply:
👍 via 6280997